src/modules/crud.js

```diff
@@ -146,10 +153,46 @@ export const showEditInput = (paregraphElement) => {
   input.focus();
 };
 
+// eslint-disable-next-line no-unused-vars
+const removeTodo = (todoId, clear = false) => {
+  let currentList = getLocal();
+  if (!clear) {
+    currentList = currentList.filter((todo) => todo.id !== todoId);
+  } else {
+    currentList = currentList.filter((todo) => todo.id !== true);
+  }
+  updateLocal(currentList);
+};
+
+list.addEventListener('click', (e) => {
+  if (e.target.tagName === 'INPUT') {
+    const checkbox = e.target.checked;
+    const currentList = getLocal();
+    const task = currentList.find((todo) => todo.index === Number(e.target.id));
+    task.completed = checkbox;
+    updateLocal(currentList);
+  }
+});
+
+const clearCompleted = () => {
+  let currentList = getLocal();
+
+  currentList = currentList.filter((todo) => !todo.completed);
+  currentList = currentList.map((todo, index) => {
+    todo.index = index + 1;
+    return todo;
+  });
+  updateLocal(currentList);
+  loadTodo();
+};
```

Comment:
- [ ] I've noticed you've added more functionality from the previous review but unfortunately, I cannot test your update and "clearCompleted" functionality as there is no data on the page and there is no way for me to add task to the todo list. please fix this issue so that it is possible to test the completed and updated functionality 🙏.